src/middlewared/middlewared/plugins/pool_/import_pool.py

```diff
@@ -224,6 +227,111 @@ async def import_pool(self, job, data):
 
         return True
 
+    @api_method(PoolReimportArgs, PoolReimportResult, roles=['POOL_WRITE'])
+    @job(lock='pool_reimport')
+    async def reimport(self, job, oid):
+        """
+        Attempt to import a pool that exists in database but is currently OFFLINE.
+
+        This is useful after SED disks have been unlocked and the pool can now be imported.
+
+        .. examples(websocket)::
+
+          Reimport pool of id 1 after unlocking SED disks.
+
+            :::javascript
+            {
+                "id": "6841f242-840a-11e6-a437-00e04d680384",
+                "msg": "method",
+                "method": "pool.reimport",
+                "params": [1]
+            }
+        """
+        pool = await self.middleware.call('pool.get_instance', oid)
+        vol_name = pool['name']
+        vol_guid = pool['guid']
+
+        verrors = ValidationErrors()
+        if pool['status'] != 'OFFLINE':
+            verrors.add(
```

Comment:
Caleb said we're moving away from validation errors (look at how new ZFS plugins are designed). You can just throw CallError with corresponding errno now.